src/test/java/org/opentripplanner/netex/NetexLoaderSmokeTest.java

```diff
@@ -24,33 +45,146 @@
     private static final String NETEX_FILENAME = "netex_minimal.zip";
 
     @Test
-    public void smokeTestOfNetexLoadData() throws Exception {
+    public void smokeTestOfNetexLoadData() {
```

Comment:
I have changed this test to do proper assertions (not just on size of collections).